canonical-web-and-design/dqlite.io
# Add static docs

Added the docs from the upstream project to the site. I will propose moving the docs to this project temporarily until discourse can host them.

 ## QA
- Load /docs
- Check it looks ok
- Check the other pages load ok
